plugins/a11yhelp/dialogs/a11yhelp.js

```diff
@@ -139,6 +132,10 @@ CKEDITOR.dialog.add( 'a11yHelp', function( editor ) {
 				var item = items[ j ],
 					itemLegend = item.legend.replace( variablesPattern, replaceVariables );
 
+				// (#16980) There should be a different hotkey shown in Commands on Edge browser.
+				if ( section.name === 'Commands' && ( j === 10 && CKEDITOR.env.edge || j === 11 && !CKEDITOR.env.edge ) )
```

Comment:
Here you have used so called "magic numbers", usage of which would backfire to us very quickly. This is not a desired way to do this.